script_editor/app/views/plays/show.html.erb

```diff
@@ -387,7 +421,7 @@
                 )
                 // Update selection
                 if (srcElement.className == "word" || srcElement.className == "punc" ||
-                    srcElement.className == "stage") {
+                    srcElement.className.indexOf("stage") !== -1) {
```

Comment:
Can you explain what this fixes? I don't understand this.

Reply:
Also would `srcElement.className.includes("stage")` work the same? I think its a little easier to understand than the -1.